source/extensions/filters/network/redis_proxy/conn_pool_impl.h

```diff
@@ -172,11 +174,14 @@ class InstanceImpl : public Instance {
   };
 
   struct LbContextImpl : public Upstream::LoadBalancerContextBase {
-    LbContextImpl(const std::string& hash_key) : hash_key_(std::hash<std::string>()(hash_key)) {}
+    LbContextImpl(const std::string& key, bool enabled_hashtagging)
+        : hash_key_(std::hash<std::string>()(hashtag(key, enabled_hashtagging))) {}
```

Comment:
This confused me when I first saw this, so I looked at the code (the use of `std::hash` here). @HenryYYang I think this needs to get fixed also to use a configuration setting for the hash type (similar to your previous PR) or its not going to work out well when you migrate? I think this was effectively missed in the other PR. This is unrelated to the intent of this PR so no need to change here but I wanted to flag it for when we try to get rid of `std::hash` in the hashing path.